<?xml version="1.0" ?>
<testsuite errors="0" failures="0" name="django.contrib.auth.tests.forms.PasswordChangeFormTest" tests="4" time="0.003">
	<testcase classname="django.contrib.auth.tests.forms.PasswordChangeFormTest" name="test_field_order" time="0.001"/>
	<testcase classname="django.contrib.auth.tests.forms.PasswordChangeFormTest" name="test_incorrect_password" time="0.001"/>
	<testcase classname="django.contrib.auth.tests.forms.PasswordChangeFormTest" name="test_password_verification" time="0.001"/>
	<testcase classname="django.contrib.auth.tests.forms.PasswordChangeFormTest" name="test_success" time="0.001"/>
	<system-out/>
	<system-err/>
</testsuite>
